README.md

```diff
@@ -79,15 +79,6 @@ Once you have your test framework configured to write its test results to a file
     # Without it, we won't be able to find the job to associate your artifacts.
     # If you're not sure, you can always safely set this even if you aren't using a matrix.
     job-matrix: '${{ toJSON(matrix) }}'
-
-    # Optional.
-    # The behavior of the action when an artifact's file is not found.
-    #
-    #  Options:
-    #    - ignore: Output neither a warning nor an error; the action will succeed.
-    #    - warn (default): Output a warning; the action will succeed.
-    #    - error: Output an error; the action will fail.
-    if-files-not-found: ''
```

Comment:
The new behavior is such that if the files aren't found the action will hard error, right? If any customers' use of the action are misconfigured, removing ignore/warn this will break their pipelines. Otherwise LGTM

Reply:
No, it fails quietly and skips the upload (I tested this in another branch by stopping these tests from emitting JUnit artifacts)

Reply:
So we’d go from failing their build if they had error set to _not_ failing their build. 

Based on how the CLI is written, that’s at least the expected behavior of using it directly. We could do some work to support this use case though if we think it’s important

Reply:
https://github.com/rwx-research/upload-captain-artifact/actions/runs/3567381993/jobs/5995011560#step:7:20

Reply:
Ah, sorry for the misunderstanding. Thanks for explaining.

Reply:
No worries thanks for the 👀!